ansible/tasks/config.yml

```diff
@@ -0,0 +1,9 @@
+project_root: "{{ ansible_env.HOME }}/node"
+aeternity_bin: "{{ ansible_env.HOME }}/node/bin/aeternity"
+download_dir: /tmp/mnesia_backups
+backup_bucket: aeternity-database-backups
+backup_suffix: backup
+health_check: yes
+format: gz
```

Comment:
May be this should be **zip** as it is changed in snapshot playbook ?

Reply:
snapshoot should use {{ format }} fixed.